runtime/connectors/connectors.go

```diff
@@ -79,7 +79,18 @@ func (ps PropertySchema) ValidateType(val any) bool {
 type Env struct {
 	RepoDriver string
 	RepoDSN    string
-	Variables  map[string]string
+	// user provided env variables kept with keys converted to uppercase
+	//
+	// following env variables are used in connectors :
+	// gcs specific variables :
+	// - GCS_CREDENTIALS : credential file content in json format
+	//
+	// s3 specific variables :
+	// - AWS_ACCESS_KEY_ID
+	// - AWS_SECRET_ACCESS_KEY
+	// - AWS_SESSION_TOKEN
```

Comment:
Saw your other comment about keeping central for docs in future. I actually think we should just have the main docs link out to the godoc of the individual connectors. They will be the best source of truth for what config is available – having to document anywhere else, I'm sure we'll forget to add/edit keys at some point :) So I think we can remove from here.